ItemNetworthCalculator.js

```diff
@@ -0,0 +1,42 @@
+const { calculateItemNetworth } = require('./helper/calculateNetworth');
+const { parsePrices } = require('./helper/prices');
+const { validateItem } = require('./helper/validate');
+
+const networthManager = require('./NetworthManager');
+
+class ItemNetworthCalculator {
+    /**
+     * Creates a new ItemNetworthCalculator
+     * @param {object} itemData The item the networth should be calculated for
+     */
+    constructor(itemData) {
+        this.itemData = itemData;
+        validateItem(this.itemData);
+    }
+
+    /**
+     * Returns the networth of an item
+     * @param {object} [prices] A prices object generated from the getPrices function. If not provided, the prices will be retrieved every time the function is called
+     * @returns {object} An object containing the item's networth calculation
+     */
+    async getNetworth(prices) {
+        return this.#calculate(prices, false);
+    }
```

Comment:
Few ideas:

- Add arguments for every options they could override.
- Default to values from `networthManager`
- No need to call the private method `#calculate`? Just stick the logic in this method to reduce code?